e2e/dragonfly_controller_test.go

```diff
@@ -609,12 +654,20 @@ var _ = Describe("Dragonfly Server TLS tests", Ordered, FlakeAttempts(3), func()
 			TLSSecretRef: &corev1.SecretReference{
 				Name: "df-tls",
 			},
+			Authentication: &resourcesv1.Authentication{
```

Comment:
This is needed as an auth mechanism is necessary when we pass a TLS certificate.